crates/wasm-sdk/src/errors.rs

```diff
@@ -1,10 +1,6 @@
-/// Generic result
-pub type Result = ::std::result::Result<(), Error>;
-
-// TODO: error is way too big, replace w/ aliased anyhow or Box<dyn std::error::Error>
-
 /// Generic error
 pub type Error = ::anyhow::Error;
+pub use ::anyhow::anyhow as error;
```

Comment:
I think this is okay because all of the cases where `anyhow::Error` is explicitly referenced, it's wrapping that error in another error owned by the crate referencing it. Both the `bulwark_wasm_sdk` and `wasmtime` dependencies use an aliased `anyhow::Error`, which the error value can interchangeably wrap. I'm not sure what would happen if I tried to use `thiserror` to derive a `From` trait for two different aliases of the same underlying error type?

Reply:
I tried it out. It doesn't seem to be allowed:

```
conflicting implementations of trait `std::convert::From<wasmtime::Error>` for type `errors::PluginLoadError`
conflicting implementation for `errors::PluginLoadError`
```

Your conclusion that the two aliases of the same underlying type is correct, but it seems that you can't have two different `From` trait implementations for two different aliases of the same underlying type.